homeassistant/components/seventeentrack/strings.json

```diff
@@ -45,6 +45,17 @@
     "deprecated_yaml_import_issue_invalid_auth": {
       "title": "The 17Track YAML configuration import request failed due to invalid authentication",
       "description": "Configuring 17Track using YAML is being removed but there were invalid credentials provided while importing your existing configuration.\nSetup will not proceed.\n\nVerify that your 17Track credentials are correct and restart Home Assistant to attempt the import again.\n\nAlternatively, you may remove the 17Track configuration from your YAML configuration entirely, restart Home Assistant, and add the 17Track integration manually."
+    },
+    "deprecate_sensor": {
+      "title": "17Track package sensors are being deprecated",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "[%key:component::seventeentrack::issues::deprecate_sensor::title%]",
+            "description": "17Track package sensors are deprecated and will be removed.\nPlease update your automations and scripts to use `Home Assistant Core Integration: Service call`."
```

Comment:
Maybe we should explain what service to use

Reply:
added